src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -57,6 +57,7 @@ public static class EnvironmentSettingNames
         public const string RoleInstanceId = "RoleInstanceId";
         public const string HealthPingEnabled = "WEBSITE_FUNCTIONS_HEALTH_PING_ENABLED";
         public const string TestDataCapEnabled = "WEBSITE_FUNCTIONS_TESTDATA_CAP_ENABLED";
+        public const string AzureMonitorEnabled = "WEBSITE_FUNCTIONS_AZUREMONITOR_ENABLED";
```

Comment:
I'd recommend a change to this. Today we only have one log category (FunctionAppLogs), but we've already been discussing adding a second (something like FunctionInvocationResults -- that has just the success/fail results from all invocations, minus all the trace logs). 

That turns this from an on/off switch to a collection of categories, depending on how you surface this from DWAS. So I think this should change to be something like `WEBSITE_FUNCTIONS_AZUREMONITOR_CATEGORIES`. And today, if "FunctionAppLogs" is there, we add the provider.